api/audit/tasks.py

```diff
@@ -123,6 +126,7 @@ def create_segment_priorities_changed_audit_log(
     feature_segment_ids: typing.List[int],
     user_id: int = None,
     master_api_key_id: int = None,
+    changed_at: str = None,
```

Comment:
I debated a larger (and arguably more elegant) change to resolve this issue which was to modify AbstractBaseTask.run() to pass in the `Task` as a key word argument to the task function. This, however, would mean modifying all existing task functions to ensure that they can accept the additional kwarg. Since it's currently only really needed by this task (since the others have the date available from other means), I figured it was overengineering at this stage.

Reply:
I agree that this looks like a more practical solution than modifying AbstractBaseTask.run().¡ in this case, it is my personal POV